src/components/new-request/search.vue

```diff
@@ -560,13 +560,6 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin, Sear
       !this.isSupportedContinuationIn(this.getEntityTypeCd)
     ) return true
 
-    // Conditional for Continuation In Flow.
-    if (
-      this.isContinuationIn &&
-      this.isNumberedCompany &&
-      !this.isSupportedContinuationIn(this.getEntityTypeCd)
-    ) return true
-
```

Comment:
Duplicate of lines 556-561 above. (Probably a merge issue some time ago.)

Reply:
Hahahahahaha, might as well add a 3rd one. To be extra, extra, Extra sure 🤣